### PR TITLE
Remove unneeded `Juxt` token

### PR DESCRIPTION
**Pull request by neuroevolutus**
I started reading through the parsing code and noticed that this token was being unused. Let me know if you'd like me to make any further changes!

### CODE REVIEW

File: scrapscript.py [hunk 1]
```diff
@@ -394,8 +388,6 @@ def parse(tokens: typing.List[Token], p: float = 0) -> "Object":
         # we can match variants in MatchFunction
         # It needs to be higher than the precedence of the && operator so that
         # we can use #true() and #false() in boolean expressions
-        # It needs to be higher than the precedence of juxtaposition so that
```
Comment (tekknolagi) [inline scrapscript.py]: Please leave this comment; still applies

**Comment by tekknolagi** [comment]
Thanks for the fix! Please add back the comment or I'll do it in a couple of days. Then we merge!

If you would like to work on anything else, please let me know

File: scrapscript.py (hunk 1 above)
Comment (neuroevolutus) [reply scrapscript.py]: Oh, I see. Thanks for pointing that out. I've restored this hunk.

**Comment by neuroevolutus** [comment]
I saw that #84 was about utilizing Pratt parsing, but it seems like you've already implemented a pretty neat Pratt parser for the language. Are there any other parts of the parsing code that could use some help?

A couple of things I was thinking about:
* Would turning the lexer into a generator be helpful?
* What is specifically going wrong about the precedence of the spread operator?

**Comment by tekknolagi** [comment]
> I saw that #84 was about utilizing Pratt parsing, but it seems like you've already implemented a pretty neat Pratt parser for the language. Are there any other parts of the parsing code that could use some help?

84 is mostly about deciding if we want to have the lexer be a generator and have the parser pull from it. But I also dropped a Pratt parsing resource because our parser is a big ball of hacks and I wonder if it's possible to make it more regular-looking.

>     * What is specifically going wrong about the precedence of the spread operator?

That is a good question. I think it feels too low in the precedence list but didn't have any actual notion of what that means or why it's wrong. Added in 33a6c8f3 but not explained well.

**Comment by tekknolagi** [comment]
Thank you for the PR!

**Pull request merged**